docs/standard/base-types/snippets/regular-expression-example-scanning-for-hrefs/csharp/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+public class Class1
+{
+    // <main>
+    public static void Main()
+    {
+        string inputString = "My favorite web sites include:</P>" +
+                             "<A HREF=\"https://docs.microsoft.com/en-us/dotnet/\">" +
```

Comment:
To confirm. The reason html tags and attributes are capitalized is to show that the regex expression is case insensitive?

Reply:
No, probably just old style............................................................................